src/database/standalone-sqlite.ts

```
@@ -3231,6 +3277,10 @@ if (!isMainThread) {
           const maxHeight = worker.getMaxHeight();
           parentPort?.postMessage(maxHeight);
           break;
+        case 'getMaxStableBlockTimestamp':
+          const maxStableBlockTimestamp = worker.getMaxStableBlockTimestamp();
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)